.github/workflows/cross-compiles.yml

```diff
@@ -150,7 +167,7 @@ jobs:
             tests: none
           }
         ]
-    runs-on: ${{ github.server_url == 'https://github.com' && 'ubuntu-latest' || 'ubuntu-22.04-self-hosted' }}
+    runs-on: ${{ github.server_url == 'https://github.com' && 'ubuntu-24.04' || 'ubuntu-22.04-self-hosted' }}
```

Comment:
Is `ubuntu-latest` not pointing to `ubuntu-24.04`?

Also, the fallback to Ubuntu 22.04 is pointless because it will always trigger a failing build.

Reply:
> Is `ubuntu-latest` not pointing to `ubuntu-24.04`?

The README of https://github.com/actions/runner-images shows ubuntu-latest points to ubuntu-22.04 and ubuntu-24.04 is marked as "beta"

> Also, the fallback to Ubuntu 22.04 is pointless because it will always trigger a failing build.

Do not know the details of these self hosted CI. Do not know if it is safe to remove them